mem.go

```diff
@@ -2,12 +2,61 @@ package sqrl
 
 import (
 	"fmt"
+	"math/big"
 
-	"github.com/shirou/gopsutil/mem"
+	"github.com/dmmcquay/likexian/host-stat-go"
+	"github.com/dmmcquay/sqrl/dustin/go-humanize"
 )
 
-func MemInfo() {
+func meminfo() (uint64, uint64) {
 	v, _ := mem.VirtualMemory()
+	var total uint64
```

Comment:
It would be more go-like if you just did:

total := v.Total

Basically, an extra line of code that isn't needed.